### PR TITLE
Add new mentor to Python track: joaoluizn

### PR DESCRIPTION
**Pull request by joaoluizn**
1. Adding New Python Track Mentor.
2. Indentation repair over the _j.json_ file

### CODE REVIEW

**Comment by joaoluizn** [comment]
Just figure out a better approach to amend new changes without force pushing, so forgive me about this. won't happen again.

**Comment by joaoluizn** [comment]
Hm.. I'm pretty sure its 2 spaces indented. If that is wrong, so the whole file is wrong

image

**Comment by joaoluizn** [comment]
I just followed the guidelines here:

image

**Comment by rpottsoh** [comment]
Yes, there are a couple indentation issues.  You didn't cause them, but since we are working on this file, lets fix'em.  I'll be posting more comments shortly to point out what to fix.

**Comment by rpottsoh** [comment]
Line 2 should be removed.

File: mentors/python/j.json [hunk 1]
```diff
@@ -25,12 +25,12 @@
...
-     "link_url": "https://www.extension.harvard.edu/faculty-directory/jeff-parker",
-     "avatar_url": null,
-     "bio": "I enjoy the simplicity of Python."
+    "github_username": "jeffdparker",
+    "name": "Jeff Parker",
+    "link_text": "Game face",
+    "link_url": "https://www.extension.harvard.edu/faculty-directory/jeff-parker",
+    "avatar_url": null,
+    "bio": "I enjoy the simplicity of Python."
    },
```
Comment (rpottsoh) [inline mentors/python/j.json]:
```suggestion
  },
```

Comment (rpottsoh) [inline mentors/python/j.json]: Remove 1 space
```suggestion
  },
```

**Comment by rpottsoh** [comment]
Very minor things.  You took care of the bigger issue earlier.  👍

**Comment by joaoluizn** [comment]
Oh, just saw both minor changes you told. Will patch then now, I thought it was something I changed, missed this space, Sorry!

**Pull request merged**